src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java

```diff
@@ -114,17 +112,15 @@ private void testSubmitAction() {
                 BackDoor.getFeedbackResponse(fq.getId(),
                     "IFQSubmitUiT.instr@gmail.tmt",
                     "IFQSubmitUiT.instr@gmail.tmt").getResponseDetails().getAnswerString());
-        
-        submitPage.verifyHtmlMainContent("/instructorFeedbackQuestionSubmitPageFilled.html");
```

Comment:
From looking through, the backdoor accesses are already checking for the changes expected, there doesn't seem to be a need for the HTML verification tests unless it was meant for testing something else specific?